### ISSUE TITLE
Individually selected expense rows in grouped reports don't have a background color applied to them

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#45190]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.6-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @shawnborton 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1720629341208869

## Action Performed:
Precondition: User have some shared group expenses in a report
1. Go to staging.new.expensify.com
2. Click Search in bottom navigation
3. Click "Shared" > Click check box on any one of the grouped expenses
## Expected Result:
Selected expense in a group, should get a background color the same way that it does if it's a single expense row that isn't grouped is selected.
## Actual Result:
When you select an expense in a group, it doesn't get a background color the same way that it does if it's a single expense row that isn't grouped.
## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos
![CleanShot 2024-07-10 at 18 34 51@2x](https://github.com/Expensify/App/assets/38435837/36e16b5f-370d-4b5c-97a5-1096a1d59d95)

![CleanShot 2024-07-10 at 18 37 50@2x](https://github.com/Expensify/App/assets/38435837/4b7e975e-a119-451d-8f2e-d29d58d964e6)

https://github.com/Expensify/App/assets/38435837/91bc1c5d-8cb4-4fd1-a216-b3f6e7a00b8f

Add any screenshot/video evidence

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shawnborton** [comment]
> We can remove padding from parent container and apply it to child items.

Yup, I agree with this - we'd want it to look like this mockup here: 
![image](https://github.com/Expensify/App/assets/2319350/8104c619-3767-43d1-a8c2-1177fccc51a9)

**Comment by isabelastisser** [comment]
Hey @ishpaul777, can you please review the proposals above? Thanks!

**Comment by shawnborton** [comment]
Thanks! Looking now 👀

**Comment by isabelastisser** [comment]
I will be OOO tomorrow and next week, so I am reassigning this until I return on July 29. Thanks, @sakluger!

Status: PR Review/merge.

**Comment by isabelastisser** [comment]
I'm back, taking this. Thanks, @sakluger !

**Comment by isabelastisser** [comment]
The payments were processed in Upwork.

**Comment by isabelastisser** [comment]
@ishpaul777, can you please complete the BZ checklist before we can close this issue? Thanks!

**Comment by isabelastisser** [comment]
All set!
